packages/paypal-commerce-integration/src/paypal-commerce-alternative-methods/paypal-commerce-ratepay-payment-strategy.ts

```diff
@@ -74,8 +89,13 @@ export default class PaypalCommerceRatepayPaymentStrategy implements PaymentStra
             );
         }
 
+        const bodyElement = document.getElementsByTagName('body')[0];
+        bodyElement.setAttribute('id', LOADING_CONTAINER_ID);
```

Comment:
Is it possible to avoid updating body element with anything?

Reply:
unfortunately we don't have any element that guarantee will be on page and would have id and would satisfy our conditions to attach preloader indicator there